CMakeLists.txt

```diff
@@ -479,6 +479,25 @@ ELSE()
 ENDIF()
 
 
+# Find the libdap package so that Aspect can connect to the OPeNDAP servers
+# Author: Kodi Neumiller
+# Added: 10/16/18
+IF(LIBDAP)
```

Comment:
what does this IF do?

Reply:
We put this in there in case the user wants to compile aspect without libdap. We added an option for when cmake is run: 'LIBDAP=ON'. The default is set to OFF, but if the user wishes to search for and use the libdap package they will need to run cmake with the option '-DLIBDAP=ON'.

This is also used by the code we added in utilities.cc in the url_reader pull request.

Reply:
oh, okay. Can you use something like ASPECT_WITH_LIBDAP and create a cache variable like https://github.com/geodynamics/aspect/blob/master/CMakeLists.txt#L116-L118 ? This way it can be toggled inside "ccmake".

Reply:
That looks like a much better idea, I'll make those changes.